modules/seqexec/server/src/main/scala/seqexec/server/gems/GemsController.scala

```diff
@@ -29,8 +32,9 @@ object GemsController {
     odgw1: Boolean,
     odgw2: Boolean,
     odgw3: Boolean,
-    odgw4: Boolean
+    odgw4: Boolean,
+    useOI: Boolean,
+    useP1: Boolean
```

Comment:
GeMS was designed to use F2 OI, but never commissioned. Maybe it should be removed in the future. GeMS with the new NGS2 will use P1 for focus correction.